src/components/chart/common-chart-options.js

```diff
@@ -223,6 +224,25 @@ class CommonChartOptions {
         }
     };
 
+    formatAxisLabels = (chart) => {
+        const targetAxis = chart.userOptions.chart.type === 'bar' ? chart.yAxis : chart.xAxis;
+        targetAxis.forEach((axis) => {
+            axis.update({
+                labels: {
+                    useHTML: true,
+                    formatter: function () {
```

Comment:
Query: do we need to do this in a loadEvent? If we set `useHTML` true in the options (https://api.highcharts.com/highcharts/xAxis.labels.useHTML) could we not then use CSS to style the HTML element as we want?

Reply:
And a further query: do we definitely need this extra styling? Highcharts will automatically use an ellipsis to hide some of the label if it overlaps - but maybe this adds some improvements to the auto-behaviour?

Reply:
Yeah i can add the css but with the ellipsis, It wasnt working as expected when set in the options